### PR TITLE
Use the latest localstack 0.11.5 for testing Amazon Services

### PR DESCRIPTION
**Pull request by oscerd**

### CODE REVIEW

**Review by gsmet — APPROVED** [review]

**Comment by oscerd** [comment]
Amazon-lambda integration tests don't use localstack as far as I see.

**Comment by gsmet** [comment]
A related test is failing:
```
Error:  io.quarkus.it.amazon.AmazonSqsITCase.testSendAndReceiveMessage(String)[2]  Time elapsed: 30.265 s  <<< FAILURE!
java.lang.AssertionError: 
1 expectation failed.
Response body doesn't match expectation.
Expected: is "Quarkus is awsome"
  Actual: Quarkus Quarkus is awsome
```

**Comment by oscerd** [comment]
> A related test is failing:
> 
> ```
> Error:  io.quarkus.it.amazon.AmazonSqsITCase.testSendAndReceiveMessage(String)[2]  Time elapsed: 30.265 s  <<< FAILURE!
> java.lang.AssertionError: 
> 1 expectation failed.
> Response body doesn't match expectation.
> Expected: is "Quarkus is awsome"
>   Actual: Quarkus Quarkus is awsome
> ```

I'll have a look

**Comment by oscerd** [comment]
Frankly I have no more clue. It seems totally non-deterministic.

**Comment by gastaldi** [comment]
I rebased it for you

**Comment by gastaldi** [comment]
Tests are still failing. Also it looks like 0.11.6 is out, maybe update to that version too?

**Comment by oscerd** [comment]
> Tests are still failing. Also it looks like 0.11.6 is out, maybe update to that version too?

I'll try, but maybe I'm doing something wrong from my side :-)

File: docs/src/main/asciidoc/amazon-kms.adoc [hunk 1]
```diff
@@ -41,7 +41,7 @@ The easiest way to start working with KMS is to run a local instance as a contai
 
 [source,shell,subs="verbatim,attributes"]
 ----
-docker run --rm --name local-kms --publish 8011:4599 -e SERVICES=kms -e START_WEB=0 -d localstack/localstack:0.11.1
+docker run --rm --name local-kms --publish 8011:4599 -e SERVICES=kms -e START_WEB=0 -d localstack/localstack:0.11.5
```
Comment (gastaldi) [inline docs/src/main/asciidoc/amazon-kms.adoc:44]: I wonder if we can define the version in a common place for reuse?

Comment (oscerd) [reply docs/src/main/asciidoc/amazon-kms.adoc:44]: it would make sense.

**Comment by gsmet** [comment]
I tried 0.11.6 and got a load of failures. Looks like their micros are not really micros...

**Comment by gastaldi** [comment]
> I tried 0.11.6 and got a load of failures. Looks like their micros are not really micros...

Damn. How about closing this one and sticking to 0.11.1 for now? Reopen if the tests pass in your org branch?

**Comment by oscerd** [comment]
Sure thing, I'll try.

**Pull request closed without merging**